server/src/test/java/org/elasticsearch/search/sort/FieldSortIT.java

```diff
@@ -1473,6 +1487,22 @@ public void testNestedSort() throws IOException, InterruptedException, Execution
         assertThat(hits[0].getSortValues()[0], is("bar"));
         assertThat(hits[1].getSortValues()[0], is("abc"));
 
+        try {
```

Comment:
Can you use `LuceneTestCase#expectThrows` for the check below in order to ensure that an exception is thrown ?